session1/examples/interface.js

```diff
@@ -0,0 +1,23 @@
+"use strict";
+exports.__esModule = true;
+var firstName = "John";
+var lasttName = "Doe";
+var age = 22;
+var address = {
+    apartment: 101,
+    street: "Fort Street",
+    city: "Mumbai",
+    pin: 400001,
+    state: "Maharashtra"
+};
+var getAddress = function (address) {
```

Comment:
Why are using ES5 function syntax here but not the ES6 arrow function?

Reply:
TBD